nodes/core/core/20-inject.html

```diff
@@ -183,7 +184,8 @@ <h3>Details</h3>
             payloadType: {value:"date"},
             repeat: {value:"", validate:function(v) { return ((v === "") || (RED.validators.number(v) && (v >= 0))) }},
             crontab: {value:""},
-            once: {value:false}
+            once: {value:false},
+            onceDelay: {value:""}
```

Comment:
we should now expose this - as the default in the node is 100 mS so the default value should be 0.1

Reply:
OK